Assets/MixedRealityToolkit.SDK/Features/UX/Scripts/Utilities/FocusHandlerTrigger.cs

```diff
@@ -0,0 +1,38 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using Microsoft.MixedReality.Toolkit.Input;
+using UnityEngine;
+using UnityEngine.Events;
+
+namespace Microsoft.MixedReality.Toolkit.UI
```

Comment:
might want to move this input SDK/Features/Input/Handlers since the other event handlers are there.

Reply:
Good idea!